frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -51,16 +57,11 @@ export const getPipelineRunData = (
             },
     },
     spec: {
+      ...(params && { params }),
```

Comment:
```suggestion
      params,
```
You're always giving this an array. (I'd move this also back to where it was for cleaner diffs)

`getPipelineRunParams` always returns an array (`.map`) and empty arrays are truthy.

Reply:
Updated.